bundles/org.openhab.binding.mihome/README.md

```diff
@@ -48,6 +48,9 @@ After setup, you can disconnect the gateway from the internet to keep your senso
         1.  Select your Gateway in the MiHome app
         2.  Go to the "..." menu on the top right corner and click "About"
         3.  Tap the version number "Plug-in version : 2.XX.X" at the bottom of the screen repeatedly until you enable developer mode
+		NOTE: In newest version > 3.x.x
```

Comment:
I am not really sure if that renders properly on the webpage. Could you maybe add the note simply in brackets on step number 3?